jetty9/src/main/docker/50-jetty.bash

```diff
@@ -19,9 +19,25 @@ if [ ! -e "$ROOT_DIR" -a -d /app ]; then
   ln -s /app "$ROOT_DIR"
 fi
 
-# move command line args to $JETTY_ARGS
-export JETTY_ARGS="${@/$1/}"
-set - "$1"
+# If a webapp root directory exist, use it as the work directory
+if [ -d "$ROOT_DIR" ]; then
+  cd "$ROOT_DIR"
+fi
+
+# TODO update docker-entrypoint.bash to do this before calling setup scripts
```

Comment:
It needs to be done before we process the jetty args. So `docker-entrypoint.bash` needs to be modified to do this before it calls the setup scripts.....  or we have to come up with a neater way of distinguishing between:
- no args, so extend and run the command line as specified by default in the `Dockerfile`
- non executable args (eg. `--list-config`) - intended to extend the default `Dockerfile` command line, but we have lost that so have to recreate, extend then run
- executable args (eg. `ls /var/lib/jetty`) - intended to entirely replace the `Dockerfile` command.  

It currently works as is, but is a bit fragile.     However, I think it probably should be the subject of a different PR to properly fix.